alerts/tahoma.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "Tahoma integration incompatible with firmware 2020.3.4-12"
+created: 2020-07-03 12:00:00
+updated: 2020-07-03 12:00:00
+integrations:
+  - tahoma
+github_issue: https://github.com/home-assistant/core/issues/37291
+homeassistant: ">0.49"
```

Comment:
Not sure about this value...

Reply:
Should be 0.59 as that was the version the integration got introduced https://www.home-assistant.io/integrations/tahoma/
```suggestion
homeassistant: ">0.59"
```